releases/v20.1.11.md

```diff
@@ -0,0 +1,81 @@
+    <a href="https://binaries.cockroachdb.com/cockroach-v20.1.11.src.tgz"><button id="source" data-eventcategory="source-release-notes">Source</button></a>
+</div>
+
+### Docker image
+
+{% include copy-clipboard.html %}
+~~~shell
+$ docker pull cockroachdb/cockroach-unstable:v20.1.11
+~~~
+
+
+### SQL language changes
+
+- Fix a bug in [`RESTORE`](../v20.1/restore.html) where some unusual range boundaries in interleaved tables caused an error. [#58260][#58260]
+
+### Bug fixes
+
+- In v20.1.8, we attempted to fix `age`'s lack of normalization of `H:M:S` into the years, months and days field. However, this was also broken for values > 1 month, as well as breaking `a::timestamp(tz) - b::timestamp(tz)` operators. This has now been resolved. [#57956][#57956]
+- Fixed an assertion error caused by some DDL statements used in conjunction with common table expressions (`WITH`). [#57952][#57952]
+- Fixed a bug that caused temp tables to not be cleaned up after the associated session was closed. [#58167][#58167]
+- Added a safeguard against crashes while running `SHOW STATISTICS USING JSON`, which is used internally for statement diagnostics, and [`EXPLAIN ANALYZE (DEBUG)`](..v20.1/explain-analyze.html). [#58264][#58264]
+- Previously, CockroachDB could return non-deterministic output when querying the `information_schema.statistics` virtual table (internally used by the `SHOW INDEXES` command)—namely, the implicit columns of the secondary indexes could be in arbitrary order. This is now fixed, and the columns will be in the same order as they are in the primary index. [#58215][#58215]
+- Previously, CockroachDB could crash when performing a [`DELETE`](..v20.1/delete.html) operation after an alteration of the primary key when in some cases, and now it is fixed. The bug was introduced in v20.1. [#58267][#58267]
+- Fixed a panic in protobuf decoding. [#58861][#58861]
+- Fixed a bug that caused errors when accessing a tuple column (`tuple.column` syntax) of a tuples that could be statically determined to be null. [#58899][#58899]
+- Fixed an internal error involving string literals used as arrays. [#59066][#59066]
+- GC jobs now populate the `running_status` column for [`SHOW JOBS`](..v20.1/show-jobs.html). This bug has been present since v20.1. [#59138][#59138]
+- Fixed a bug in which some non-conflicting rows provided as input to an [`INSERT ... ON CONFLICT DO NOTHING`](..v20.1/insert.html) statement could be discarded, and not inserted. This could happen in cases where the table had one or more unique indexes in addition to the primary index, and some of the rows in the input conflicted with existing values in one or more unique index. This scenario could cause the rows that did not conflict to be erroneously discarded. This is now fixed. [#59172][#59172]
+
+### Contributors
+
+This release includes 24 merged PRs by 17 authors.
+We would like to thank the following contributors from the CockroachDB community:
```

Comment:
It is expected that the content below this line isn't showing up in the TeamCity preview?